### ISSUE TITLE
Copy-pasted docs means many derives are still undocumented!

### ISSUE THREAD

**Issue by TheLostLambda** [JelteF/derive_more#348]
A blocker for 1.0.0, I'd imagine, but shouldn't be too hard to fix!

I found Not / Neg first: https://docs.rs/derive_more/1.0.0-beta.6/derive_more/derive.Neg.html

But it seems to be a broader trend when things are part of the same feature!
https://docs.rs/derive_more/1.0.0-beta.6/derive_more/derive.BitAnd.html
... and presumably more!

**Comment by JelteF** [comment]
Yeah, that's indeed the case. I don't think it's worth the effort to provide specific docs for each, but it would probably be good to prefix those docs with something like: 

> Neg is derived in the same way as Not, which is described below

**Comment by tyranron** [comment]
Not a blocker, though.

**Comment by JelteF** [comment]
@TheLostLambda feel free to provide a PR to update the docs
